helm/alfresco-content-services/tests/search_test.yaml

```diff
@@ -175,3 +176,67 @@ tests:
           path: data.SEARCH_FLAVOR
           value: solr6
         template: config-infrastructure.yaml
+
+  - it: Should set deafult correct credentials for elastisearch
+    values: *testvalues
+    template: secret-search.yaml
+    asserts:
+      - equal:
+          path: data.SEARCH_USERNAME
+          value: ""
+      - equal:
+          path: data.SEARCH_PASSWORD
+          value: ""
+
+  - it: Should set correct credentials with local audit and external elastisearch
+    values: *testvalues
+    set:
+      global:
+        search:
+          username: elastic
+          password: changeme
+    asserts:
+      - equal:
+          path: data.SEARCH_USERNAME
+          value: ZWxhc3RpYw==
+        template: secret-search.yaml
+      - equal:
+          path: data.SEARCH_PASSWORD
+          value: Y2hhbmdlbWU=
+        template: secret-search.yaml
+      - equal:
+          path: data.AUDIT_ELASTICSEARCH_USERNAME
+          value: ""
+        template: secret-audit-elasticsearch.yaml
+      - equal:
+          path: data.AUDIT_ELASTICSEARCH_PASSWORD
+          value: ""
+        template: secret-audit-elasticsearch.yaml
+
+  - it: Should set correct credentials with external audit and external elastisearch as the same cluster
+    values: *testvalues
+    set:
+      global:
+        search:
+          username: elastic
+          password: changeme
+        auditIndex:
+          username: elastic
+          password: changeme
```

Comment:
maybe use different creds in the test so it doesn't accidentally pass in case we mistakenly swap them somehow